src/app/address/controllers/address/manual.test.js

```diff
@@ -67,6 +67,7 @@ describe("address controller", () => {
         addressHouseName: "My buildng name",
         addressStreetName: "street road",
         addressLocality: "small town",
+        addressCountry: "GB",
```

Comment:
```suggestion
```

This should be removed from `addressToSave` since we want to test that the value is hardcoded and it doesn't come from the form.